rpcsx-os/orbis-kernel-config/orbis-config.hpp

```diff
@@ -103,6 +103,8 @@ inline uint64_t readRegister(void *context, RegisterId id) {
     return c->gregs[REG_RSP];
   case RegisterId::rflags:
     return c->gregs[REG_EFL];
+  default:
```

Comment:
Do not use default when you do switch/case on enum. If not all variants of enum handled in switch (e.g. you have added new value to enum), you will get warning about it, if default case exists, code will silently return 0 here

Reply:
@DHrpcs3 understood, but using default + std::abort() works, right ?
I think the main idea here is to perform some action that indicates a non expected flow

Reply:
@DHrpcs3 done!